infra-as-code/bicep/modules/vwanConnectivity/vwanConnectivity.bicep

```diff
@@ -20,32 +17,44 @@ param parAzFirewallTier string = 'Standard'
 @sys.description('Switch to enable/disable Virtual Hub deployment.')
 param parVirtualHubEnabled bool = true
 
-@sys.description('Switch to enable/disable VPN Gateway deployment.')
-param parVpnGatewayEnabled bool = true
-
-@sys.description('Switch to enable/disable ExpressRoute Gateway deployment.')
-param parExpressRouteGatewayEnabled bool = true
-
-@sys.description('Switch to enable/disable Azure Firewall deployment.')
-param parAzFirewallEnabled bool = true
-
 @sys.description('Switch to enable/disable Azure Firewall DNS Proxy.')
 param parAzFirewallDnsProxyEnabled bool = true
 
 @sys.description('Prefix Used for Virtual WAN.')
 param parVirtualWanName string = '${parCompanyPrefix}-vwan-${parLocation}'
 
 @sys.description('Prefix Used for Virtual WAN Hub.')
-param parVirtualWanHubName string = '${parCompanyPrefix}-vhub-${parLocation}'
+param parVirtualWanHubName string = '${parCompanyPrefix}-vhub'
+
+@sys.description('Prefix Used for Virtual WAN Hub.')
```

Comment:
this needs extensive updating to talk about whats expected

Reply:
@jtracey93 , do you mean updating the README.md to include and explain the changes we did and how it works?

Reply:
Nope @aarunraaj line 29 is incorrect, and needs a lot more detail about the array and expected properties. You can use multi-line in the decorator as per https://learn.microsoft.com/azure/azure-resource-manager/bicep/data-types#multi-line-strings.

Please update the description for this parameter

Reply:
thanks @jtracey93 ! updated the parameters to include complete details.